README.md

```diff
@@ -22,6 +22,12 @@ Alternatively binaries are available from SourceForge at <http://sourceforge.net
 You are encouraged to review the release notes ([releasenotes.txt](releasenotes.txt)), particularly when upgrading to a 
 later version. The release notes will generally document any breaking changes.
 
+Nightly Development Builds
+--------------------------
+
+The quickest way to get the latest release of NHibernate is to add it to your project using 
+NuGet from MyGet feed (<https://www.myget.org/gallery/nhibernate>).
+
```

Comment:
Maybe also add note how to use it hassle free in teams. Something like:
To avoid everyone in team adding package source you can add/update `nuget.config` file in top folder of your project with the following content:
```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
	<packageSources>
		<add key="NHibernateDevBuilds" value="https://www.myget.org/F/nhibernate/api/v3/index.json" />
	</packageSources>
</configuration>
```

Reply:
Done